### PR TITLE
Added CSS in docs

### PR DESCRIPTION
**Pull request by AmrutaJayanti**
## Related Issue

#300 

## Description

I have added css in `docs` and created a readme.md file which includes the tutorials to be added under CSS.
I added the basic-syntax and selectors from css

## Type of PR

- [ ] Bug fix
- [ ] Feature enhancement
- [ ] Documentation update
- [x] Other (specify): _____Added CSS_____

## Checklist
- [x] I have performed a self-review of my code.
- [x] I have read and followed the Contribution Guidelines.
- [x] I have tested the changes thoroughly before submitting this pull request.
- [x] I have provided relevant issue numbers, screenshots, and videos after making the changes.
- [x] I have commented my code, particularly in hard-to-understand areas.

## Resources for Guidance

Here are some resources that may be helpful as you contribute to CodeHarborHub:
- [Code Harbor Hub Community Features](https://www.codeharborhub.live/community/features)
- [Docusaurus Documentation](https://docusaurus.io/docs/create-doc)
- [React.js Documentation](https://legacy.reactjs.org/docs/getting-started.html)
- [Markdown Guide](https://www.markdownguide.org/)
- [MDX Documentation](https://mdxjs.com/docs/)
- [Mermaid Documentation](https://mermaid.js.org/)

**Resources for DSA Features on Our Site:**

- Tabs: [Switch Languages](https://www.codeharborhub.live/community/features#multi-language-support-code-blocks)
- [Line Numbering](https://www.codeharborhub.live/community/features#line-numbering)
- [Math Equations](https://www.codeharborhub.live/community/features#math-equations-with-katex)

### CODE REVIEW

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
To maintain consistency and ease of maintenance in our content, please follow the structure outlined in our previous tutorials. Adhering to this format ensures good practice and uniformity.

**Comment by AmrutaJayanti** [comment]
@Ajay-Dhangar Can I create another folder which has CSS-basics in docs/CSS and add these in it. So that other CSS related like CSS animations, About bootstrap could be added later

**Comment by Ajay-Dhangar** [comment]
> @Ajay-Dhangar Can I create another folder which has CSS-basics in docs/CSS and add these in it. So that other CSS related like CSS animations, About bootstrap could be added later

Yes, absolutely! We need to create superior content to stand out among the best sites out there. Our goal is to provide valuable content that is not only high-quality but also free of charge and ad-free, setting us apart from the competition.

**Comment by AmrutaJayanti** [comment]
@Ajay-Dhangar  I made the changes. Please approve

File: docs/CSS/readme.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+1. Inline CSS:
+
+Applied directly to an HTML element using the style attribute.
+```html
+<p style="color: blue; font-size: 14px;">Hello World</p>
+```
+
+2. Internal CSS:
+
+Defined within a <style> tag in the HTML document's <head>.
```
Comment (Ajay-Dhangar) [inline docs/CSS/readme.md:39]: bad practice:

```md
 Defined within a <style> tag in the HTML document's <head>.
```

Good Practice:

```md
Defined within a `<style>` tag in the HTML document's `<head>`.
```

**Review by Ajay-Dhangar — CHANGES_REQUESTED** [review]
Now change code in `readme.md` file based on suggestion.

**Comment by AmrutaJayanti** [comment]
@Ajay-Dhangar  I made changes. Please approve

**Review by Ajay-Dhangar — APPROVED** [review]
Your contribution is appreciated, but it does not align well with the structure and value of my site’s content. Next time, please improve the quality of your content for better understanding and greater value.

**Pull request merged**